api/build.js

```diff
@@ -135,6 +135,29 @@ export async function build({ state, config, cwd = process.cwd() }) {
       sourcemap: false,
     });
 
+    // build hydration support script
+    await esbuild.build({
+      entryPoints: [
+
+        // TODO: can't use require.resolve here as it will resolve in a Node context and get the wrong script
+        // probably need to create a file that does import "@lit-labs/ssr-client/lit-element-hydrate-support.js";
+        // and use that as input to entryPoints
```

Comment:
An option might be to replace the parts of the path returned by `require.resolve` to look like what we need. Looks like from `package.json` that a `.replace("/node", "")` could do the trick. Maybe a bit brittle though.

Reply:
I'm also wondering if there's not some way to tell require.resolve that we are in a browser context or something. Esbuild and Rollup get it right when you use those so there's maybe something to that though I've no idea how yet